apps/worker/src/app/workflow/usecases/send-message/send-message.base.ts

```diff
@@ -48,6 +49,19 @@ export abstract class SendMessageBase extends SendMessageType {
     super(messageRepository, createExecutionDetails);
   }
 
+  protected combineOverrides(
+    bridgeData: Record<string, any> | null | undefined,
+    overrides: Record<string, any> | undefined,
+    stepId: string | undefined,
+    integrationId: string
+  ): Record<string, unknown> {
+    const bridgeProviderData = bridgeData?.providers?.[integrationId] || {};
+    const workflowGlobalProviderOverrides = overrides?.providers?.[integrationId] || {};
+    const triggerOverrides = stepId ? overrides?.steps?.[stepId]?.providers[integrationId] || {} : {};
+
+    return merge({}, bridgeProviderData, workflowGlobalProviderOverrides, triggerOverrides);
```

Comment:
We deeply merge all levels:

- Bridge
- Trigger Workflow
- Rrigger Step specific

in that order